app/views/shared/components/_course_lessons.html.erb

```diff
@@ -4,41 +4,38 @@
       <h2 class="text-black0 text-base font-semibold">
         <%= course_module.title %>
         <span class="text-primary pl-5 text-sm font-medium">(<%= lessons_count(course_module) %>)</span>
-        <!-- admin actions start -->
-        <% if policy(:course_module).show? %>
-          <%= link_to "Edit module", course_module_path(course, course_module), class: "link-text" %>
-        <% end %>
-
-        <% if policy(:course_module).moveup? %>
-          <%= link_to "Move up", moveup_course_module_path(course, course_module), data: { turbo_method: :put }, class: "link-text" %>
-        <% end %>
-
-        <% if policy(:course_module).movedown? %>
-          <%= link_to "Move down", movedown_course_module_path(course, course_module), data: { turbo_method: :put }, class: "link-text" %>
-        <% end %>
-        <!-- admin actions end -->
       </h2>
-      <div class="flex gap-4">
+      <div class="flex gap-4 items-center">
         <% if policy(@course).edit? %>
           <%= link_to course_module_path(course, course_module), class: "link-text" do %>
             <%= render "shared/components/button_default_small", label: "Edit", icon_name: "icon-edit" %>
           <% end %>
         <% end %>
-        <div class="flex flex-col gap-1 items-center">
+        <div class="flex flex-col items-center">
+          <% if policy(:course_module).moveup? %>
+
+          <%= link_to moveup_course_module_path(course, course_module), data: { turbo_method: :put }  do%>
           <button>
             <span class="icon icon-arrow-up icon-small block"></span>
           </button>
+          <% end %>
+          <% end %>
+          <% if policy(:course_module).movedown? %>
+
+          <%= link_to movedown_course_module_path(course, course_module), data: { turbo_method: :put } do%>
           <button>
             <span class="icon icon-arrow-down icon-small block"></span>
           </button>
+          <% end %>
+          <% end %>
+
         </div>
 
         <div class="flex items-center">
```

Comment:
What are these icons doing here ?

Reply:
For moving up and down

Reply:
button tag removed and span is given inside the link